packages/instant/src/util/big_number.ts

```diff
@@ -0,0 +1,29 @@
+import { BigNumber } from '@0x/utils';
+import * as _ from 'lodash';
+
+/**
+ *  A BigNumber extension that is more flexible about decimal strings.
+ *  Such as allowing:
+ *  new BigNumberInput(0.) => 0
+ *  new BigNumberInput(1.) => 1
+ *  new BigNumberInput(1..) => still throws
+ */
+export class BigNumberInput extends BigNumber {
+    private readonly _hasDecimalPeriod: boolean;
+    constructor(bigNumberString: string) {
+        const hasDecimalPeriod = _.endsWith(bigNumberString, '.');
```

Comment:
minor but would prefer this var to be more descriptive, like `isEndingWithDecimal`